### PR TITLE
Orchestration landing page doc update for launch

### PR DESCRIPTION
**Pull request by Jovonni**
updated the entrypoint guide landing page for orchestration docs from internal feedback.

### CODE REVIEW

File: main/guides/orchestration/index.md [hunk 1]
```diff
@@ -1,11 +1,20 @@
+# What is Agoric Orchestration?
+
+Agoric’s Orchestration capability allows developers to easily build cross-chain capabilities into existing applications or to create novel cross-chain-focused products.
+
+The Agoric Orchestration API sits on top of Agoric’s novel VM that provides three key elements that make multichain applications possible:
+
+- **Remote account control and transfer**: Use the Orchestration APIs to easily create accounts on remote chains, transfer assets, and invoke contracts. Your Agoric contract orchestrates all behavior directly.
```
Comment (mujahidkay) [inline main/guides/orchestration/index.md:7]:
```suggestion
- **Remote account control and transfer**: Use the Orchestration API to easily create accounts on remote chains, transfer assets, and invoke contracts. Your Agoric contract orchestrates all behavior directly.
```

File: main/guides/orchestration/index.md [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
+- **Multiblock execution with async & await**: Agoric applications communicate asynchronously and await responses which may come in the same block or many blocks (or weeks!) later. Contracts simply continue executing when the response arrives.
+- **On-chain Timers**: Contracts can set timers for regular execution which makes executing common activities like subscriptions easy.
+
+Agoric’s Orchestration APIs simplify controlling remote chains, moving assets, and using capabilities on any chain the API reaches.
```
Comment (mujahidkay) [inline main/guides/orchestration/index.md:11]:
```suggestion
Agoric’s Orchestration API simplifies controlling remote chains, moving assets, and using capabilities on any chain the API reaches.
```

**Review by mujahidkay — COMMENTED** [review]
minor suggestions for consistency

**Review by mujahidkay — COMMENTED** [review]
on a second look? I think we should match the navbar entry with this. 

Also, we need to change Orchestration Overview to an h2. Can't have two h1s.

File: main/guides/orchestration/index.md [hunk 3]
```diff
@@ -1,11 +1,20 @@
+# What is Agoric Orchestration?
+
+Agoric’s Orchestration capability allows developers to easily build cross-chain capabilities into existing applications or to create novel cross-chain-focused products.
```
Comment (amessbee) [inline main/guides/orchestration/index.md:3]: "capability" does not sound right twice in a sentence. WDYT

File: main/guides/orchestration/index.md [hunk 4]
```diff
@@ -1,11 +1,20 @@
...
+
+The Agoric Orchestration API sits on top of Agoric’s novel VM that provides three key elements that make multichain applications possible:
+
+- **Remote account control and transfer**: Use the Orchestration APIs to easily create accounts on remote chains, transfer assets, and invoke contracts. Your Agoric contract orchestrates all behavior directly.
+- **Multiblock execution with async & await**: Agoric applications communicate asynchronously and await responses which may come in the same block or many blocks (or weeks!) later. Contracts simply continue executing when the response arrives.
```
Comment (amessbee) [inline main/guides/orchestration/index.md:8]: perhaps use ticks for `async` and `await` - also maybe use "and" instead of &.

Comment (amessbee) [inline main/guides/orchestration/index.md:8]:
```suggestion
- **Multiblock execution with `async` and `await`**: Agoric applications communicate asynchronously and `await` responses which may resolve in the same block or many blocks (or weeks!) later. Your contract simply continues executing when the response arrives.
```

File: main/guides/orchestration/index.md (hunk 1 above)
Comment (Jovonni) [reply main/guides/orchestration/index.md:7]: thoughts here, I think we are speaking about the singular use of API here, the orchestration api as a single concept, not the individual apis

File: main/guides/orchestration/index.md [hunk 5]
```diff
@@ -1,11 +1,20 @@
...
+- **On-chain Timers**: Contracts can set timers for regular execution which makes executing common activities like subscriptions easy.
```
Comment (amessbee) [inline main/guides/orchestration/index.md:9]:
```suggestion
- **On-chain Timers**: Contracts can set timers for regular execution which makes scheduling execution of common activities like subscriptions easy.
```

File: main/guides/orchestration/index.md [hunk 6]
```diff
@@ -1,11 +1,20 @@
...
+
+Agoric’s Orchestration API simplifies controlling remote chains, moving assets, and using capabilities on any chain the API reaches.
```
Comment (amessbee) [inline main/guides/orchestration/index.md:11]:
```suggestion
Agoric’s Orchestration API simplifies controlling accounts on remote chains, moving assets, and using capabilities on any chain the API reaches.
```

Comment (amessbee) [inline main/guides/orchestration/index.md:11]: **any chain the API reaches** sounds very informal - it's fine if marketing is onboard with it.

**Comment by amessbee** [comment]
left a few nit comments - looks like a great improvement overall!

**Review by amessbee — APPROVED** [review]
lgtm - nice work!

**Review by mujahidkay — APPROVED** [review]
Good stuff!

**Pull request merged**